### PR TITLE
Branch specific things can now be programable decided by git-find-branch script

### PR DESCRIPTION
**Pull request by jkonecny12**
Because of our new branching policy all the code from `f**-devel` branch will be in the master branch. This is good for code but it is bad when we need specific things like `zanata` or `mock`. When we change `zanata.xml` on f25-devel the change will propagate to master branch and then we are pushing translation from master to f25 branch on `zanata`. 

New script `git-find-branch` is implemented to help us with this problem. This small script will resolve our actual branch to our main development branches (`f25-devel`, `master`, `unstable`, `rhel7-branch`...) and return the branch name without the `-*` suffix. So `f25-release` and `f25-devel` will be `f25`, this seems easier to me and this behavior can be of course changed in future. The `configure` can then substitute this variable to any file where this is needed.

### CODE REVIEW

File: scripts/git-find-branch [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#
+# Red Hat Author(s): Jiri Konecny <jkonecny@redhat.com>
+#
+#
+# There are two ways how find parent branch of this project:
+#
+# 1) When environment variable ANACONDA_TARGET_BRANCH is set, use this
+#    variable as output without any checks. This is for automated tests
+#    mainly.
+# 2) Look on the Git find all interesting branches for us. Test if
```
Comment (vpodzime) [inline scripts/git-find-branch]: "*Look at Git...*"

File: scripts/git-find-branch [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+#    actual branch is not on of those and if not look on the parent and
+#    test parent. Repeat the process with parents.
+#
+#
+# WARNING!
+# This won't work without a merge commit! Git don't have any information
```
Comment (vpodzime) [inline scripts/git-find-branch]: "*Git doesn't have...*"

File: scripts/git-find-branch [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+# which branch is the parent without the merge commit, both branches tips
```
Comment (vpodzime) [inline scripts/git-find-branch]: "*tips of both branches...*"

File: .gitignore [hunk 4]
```diff
@@ -82,3 +82,4 @@ modules.rst
 pyanaconda.*rst
 .coverage
 docs/tests/*.rst
+zanata.xml
```
Comment (vpodzime) [inline .gitignore]: I think this also needs to be added to ``MAINTAINERCLEANFILES`` in the ``Makefile.am`` (just with the ``.in`` suffix).

**Comment by jkonecny12** [comment]
**Updated:**
I hope everything will be alright now :)

**Comment by vpodzime** [comment]
> Updated:
> I hope everything will be alright now :)

It is. :)

**Pull request merged**